plugin/db/mysql/rollback.go

```diff
@@ -100,10 +101,6 @@ func (driver *Driver) GenerateRollbackSQL(ctx context.Context, binlogFileNameLis
 		return "", errors.WithMessage(err, "failed to parse binlog stream")
 	}
 
-	threadID, err := driver.GetMigrationConnID(ctx)
```

Comment:
This is a bug. The migration executing driver and rollback SQL generating driver are different. We should use the recorded thread ID of the transaction-executing driver.